Answers.md

```diff
@@ -2,10 +2,26 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+    *Answer*- React uses a virtual DOM to not only minimize the load DOM manipulation usually entails. 
+    And creates a way to re-render the entire page all the time without affecting performance. 
+    
+
+    
 1.  What does it mean to _think_ in react?
 
+    *Answer* - To think in react is to think about breaking your app down into small reusable components 
+    that also render a new UI to the page within the Code itself. 

```

Comment:
Great anwser! Here's a resource for more information about *Thinking* in React.
https://reactjs.org/docs/thinking-in-react.html